app-modules/portal/resources/views/livewire/render-knowledge-management-portal.blade.php

```diff
@@ -37,7 +37,7 @@
 @endphp
 
 <div class="flex items-center justify-center">
-    <div class="w-full max-w-full">
+    <div class="w-full max-w-6xl">
```

Comment:
Sorry, to be more clear about the ticket.

This should only be applied if the boxed portal view setting is the one selected. It is passed in through the API and currently applies some padding in the Vue component if it is selected I believe.

So we need to do this in the Vue component instead of what we are doing in there with padding now